samples/csharp/MicrosoftEduGraphSamples/Workflows/ModuleWorkflow.cs

```diff
@@ -25,12 +25,12 @@ public ModuleWorkflow(IConfiguration configuration)
         /// <summary>
         /// Workflow to create and publish the module
         /// </summary>
-        public async Task ClassworkAsync()
+        public async Task ClassworkAsync(bool appOnly = false)
```

Comment:
<param name="appOnly">True value authenticates the graph client with application permissions only, otherwise it will be created with delegated permissions</param>

Reply:
please double check, the change is not reflected

Reply:
Push operation dint happen as expected, pushed the changes to PR.

Hence resolving